vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    data_root = os.path.normpath('/home/vnath/Downloads/cellpose_dataset/')
</pre><p>To avoid using these custom paths, I suggest that you can add a "Setup data directory" section, please refer to </p><p><a href="https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb" target="_blank">https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I've updated it that the user needs to set this path manually.

Reply:
<p>Hi, the point here is that if users set this env "MONAI_DATA_DIRECTORY", then they don't need to modify the customized link you provided, I can also help modify this part. Thanks! </p>

<pre class="ql-syntax" spellcheck="false">directory = os.environ.get("MONAI_DATA_DIRECTORY")
if directory is not None:
    os.makedirs(directory, exist_ok=True)
root_dir = tempfile.mkdtemp() if directory is None else directory
print(root_dir)
</pre>